olwebsite-app/src/components/Toolbar.jsx

```diff
@@ -38,8 +38,8 @@ function Toolbar() {
           <FaSearch tabIndex="0" />
         </li>
         <li>
-          <Link to="/login">
-            <FaUser />
+          <Link to="/sign-up">
```

Comment:
Consider using NavLink just to stay consistent with the rest of the toolbar links.

Reply:
👍